specs/phase0/fork-choice.md

```diff
@@ -155,11 +168,23 @@ def get_ancestor(store: Store, root: Root, slot: Slot) -> Root:
 def get_latest_attesting_balance(store: Store, root: Root) -> Gwei:
     state = store.checkpoint_states[store.justified_checkpoint]
     active_indices = get_active_validator_indices(state, get_current_epoch(state))
-    return Gwei(sum(
+    attestation_score = Gwei(sum(
         state.validators[i].effective_balance for i in active_indices
         if (i in store.latest_messages
             and get_ancestor(store, store.latest_messages[i].root, store.blocks[root].slot) == root)
     ))
+    proposer_score = Gwei(0)
+    if store.proposer_score_boost.root != Root():
+        block_slot = store.blocks[root].slot
+        if get_ancestor(store, root, block_slot) == store.proposer_score_boost.root:
+            num_validators = len(get_active_validator_indices(state, get_current_epoch(state)))
+            avg_balance = get_total_active_balance(state) // num_validators
+            block_epoch = compute_epoch_at_slot(block_slot)
+            committee_size = get_committee_count_per_slot(state, block_epoch) * TARGET_COMMITTEE_SIZE
```

Comment:
This is not the committee size.

The committee size here (the sum of all committees for the slot) is just `num_validators // SLOTS_PER_EPOCH`. 

Committees generally very between `TARGET_COMMITTEE_SIZE` and `2*TARGET_COMMITTEE_SIZE` until you hit `MAX_COMMITTEES_PER_SLOT` and then they just grow unbounded. So we won't get a proper proposer boost just using the "target"